data/font_definitions.c

```diff
@@ -0,0 +1,284 @@
+#include "global.h"
+#include "graphics.h"
+#include "src/bitmap_font.h"
+
+
+/* FONT DEFINITIONS */
+
+
+// WarioWare Display Font: Large
+extern const u8 warioware_outline_large_symbol_glyphs_bin[]; // Textures - Punctuation/Symbols
```

Comment:
maybe make a header file for these externs? look at what i did for the stuff in games/<>/graphics/

Reply:
Done, I think.